alex-w/client/partials/index.html

```diff
@@ -0,0 +1,23 @@
+<!-- index.html in partial folder is for this table -->
+Filter by name: <input type="text" ng-model="scraper">
+<hr>
+<table>
+    <thead>
+      <th>Name</th>
+      <th>Show</th>
+      <th>Update</th>
+      <th>Delete</th>
+    </thead>
+    <tbody>
+      <tr ng-repeat="friend in friends | filter:scraper">
+        <!-- can i convert the tr below to ng-bind with two inputs? -->
+        <td>{{friend.first}} {{friend.last}}</td>
```

Comment:
I think we talked about this yesterday, but we can use concatenation inside of ng-bind.